### DISCUSSION TITLE
[Feature Request] A list along the left side with 'Favourites' or 'Bookmarks' with user defined folders

### DISCUSSION THREAD

**Discussion by Quantumatum** [filebrowser/filebrowser#2263] [Ideas]
I feel that a favourites list is missing from filebrowser, I think it would fit quite nicely along the left side column!

**Comment by bazmattaz** [comment]
Can I +1000 on this.

I have a huge file system and do a lot of moving copying/moving it would be so helpful if you could favorite a folder and it would appear on the left. Like you can do in Windows.

Please please build this

**Comment by PeteTheFirst** [comment]
This would really helpfull!

**Comment by digi-banana** [comment]
Would love that as well
